subsys/net/ip/ipv4.c

```diff
@@ -118,7 +118,8 @@ enum net_verdict net_ipv4_process_pkt(struct net_pkt *pkt)
 	enum net_verdict verdict = NET_DROP;
 
 	if (real_len != pkt_len) {
-		NET_DBG("IPv4 packet size %d pkt len %d", pkt_len, real_len);
+		NET_DBG("pkt len per hdr %d != pkt real len %d, drop",
```

Comment:
Maybe it's worth having some kind of "convention" in verdict oriented message:
"<VERDICT>: <reason>"

So here it would be: "DROP: pkt len per hdr %d != pkt real len %d"

What do you think?

Reply:
@tbursztyka 

> So here it would be: "DROP: pkt len per hdr %d != pkt real len %d"
> What do you think?

Definitely looks better, and I'm all for establishing good conventions. Will update this PR shortly. @rveerama1: I hope you'd agree that making it consistent is good idea, and can update your PR too.